fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/CacheTransactionSync.java

```diff
@@ -33,23 +34,23 @@
     
     private final String transactionDataKey;
 
-    // A callback when we hit a rollback
-    private final Runnable rolledBackHandler;
+    // Called after the transaction completes (true == committed; false == rolled back)
```

Comment:
this is a useful comment.  i wonder if it should be duplicated on the javadoc for the constructors for this class (and FHIRUserTransactionFactory